libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -55,6 +55,14 @@ namespace
 	std::mutex audit_plugin_mutex;
 	// NOLINTEND(cert-err58-cpp, cppcoreguidelines-avoid-non-const-global-variables)
 
+#if __cpp_lib_chrono >= 201907
+	// use utc_clock if it's implemented (for leap seconds)
```

Comment:
This is important because AMQP considers/accounts for leap seconds?

Reply:
We don't set AMQP-level timestamps. This is more for Elasticsearch, or whatever ends up consuming the AMQP messages. This is important because `system_clock` implements leap seconds by repeating a second. Since we use millisecond-level precision, this can result in messages appearing out of order when sorted by timestamp

Reply:
Okay. Perhaps a comment to that effect could be helpful for future readers?